main/kml_generator.py

```diff
@@ -15,7 +15,7 @@ def create_kml_file(first_location_list, second_location_list, first_folder_name
     first_location_points = create_points(first_location_list, kml_folder_list_one)
     second_location_points = create_points(second_location_list, kml_folder_list_two)
     linestrings = create_linestrings(first_location_list, second_location_list, kml_folder_linestring)
-    save_string = output_folder + "/line of sight analysis.kml"
+    save_string = output_folder + "/line_of_sight_analysis.kml"
```

Comment:
It would still be good to have this go into a dedicated output dir separate from the input data files.